jinahub/indexers/dbms/PostgreSQLIndexer/__init__.py

```diff
@@ -58,22 +69,12 @@ def size(self):
         .. # noqa: DAR201
         """
         with self.handler as postgres_handler:
```

Comment:
I am not convinced of opening/closing connection with every operation. This creates quite some overhead, especially when the database is running on a distance machine. Why not open the connection once and keep it open?